common/src/main/java/dev/endgame/Endgame.java

```diff
@@ -1,24 +1,45 @@
 package dev.endgame;
 
+import dev.architectury.registry.CreativeTabRegistry;
 import dev.architectury.registry.registries.DeferredRegister;
 import dev.architectury.registry.registries.RegistrySupplier;
 import dev.endgame.items.ExampleAnimatedItem;
 import net.minecraft.core.registries.Registries;
 import net.minecraft.resources.ResourceLocation;
 import net.minecraft.sounds.SoundEvent;
+import net.minecraft.world.item.BlockItem;
 import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.state.BlockBehaviour;
+import net.minecraft.world.level.material.Material;
+
+import java.util.function.Supplier;
 
 public class Endgame {
     public static final String MOD_ID = "endgamemod";
 
+    //Register
     public static final DeferredRegister<Item> ITEMS = DeferredRegister.create(MOD_ID, Registries.ITEM);
     public static final DeferredRegister<SoundEvent> SOUNDS = DeferredRegister.create(MOD_ID, Registries.SOUND_EVENT);
+    public static final DeferredRegister<Block> BLOCKS = DeferredRegister.create(MOD_ID, Registries.BLOCK);
 
+    //Supplier
     public static final RegistrySupplier<Item> ExampleAnimated = ITEMS.register("example_animated", () -> new ExampleAnimatedItem(new Item.Properties()));
-    public static final RegistrySupplier<SoundEvent> ExampleAnimatedSound = SOUNDS.register("example_animated", () -> SoundEvent.createFixedRangeEvent(new ResourceLocation(MOD_ID, "example_animated"), 0));
 
+    public static final CreativeTabRegistry.TabSupplier TAB = CreativeTabRegistry.create(new ResourceLocation(MOD_ID, "end"),() -> new ItemStack(ExampleAnimated.get()));
+    public static final RegistrySupplier<SoundEvent> ExampleAnimatedSound = SOUNDS.register("example_animated", () -> SoundEvent.createFixedRangeEvent(new ResourceLocation(MOD_ID, "example_animated"), 0));
+    public static final RegistrySupplier<Block> BLENDER = register("blender", () -> new Block(BlockBehaviour.Properties.of(Material.STONE).friction(0.5f).strength(1f,3f).dynamicShape().noOcclusion()), new Item.Properties().arch$tab(TAB));
```

Comment:
We should have a class for each block instead since we'll need to add logic to it. We also need to use geckolib so we need to abstract what we wrote in PR #8